libs/core/src/components/pds-popover/docs/pds-popover.mdx

```diff
@@ -0,0 +1,101 @@
+import { DocArgsTable, DocCanvas } from '@pine-ds/doc-components';
+import { components } from '../../../../dist/docs.json';
+
+# Popover
+
+Popovers display contextual information when users click or fires a keyboard event on an element. They typically contain more information than tooltips and can include interactive elements.
+
+## Guidelines
+### Usage
+- Use popovers for complex information that might overwhelm the main content.
+- Consider using a tooltip for simpler explanations or brief labels.
+
+#### When not to use
+- Don't use popovers for critical information that must be readily available.
+
+## Properties
+
+<DocArgsTable componentName="pds-popover" docSource={components} />
+
+## Variants
+### Default
+The popover's appearance and behavior are primarily controlled by two slots:
+1. **`trigger`:** This slot is used to define the element that triggers the popover. It can be any HTML element, such as a button or link.
+2. **`content`:** This slot is used to define the content of the popover. It can contain any HTML elements, including text, images, forms, or other components.
```

Comment:
```suggestion
2. **`content`:** This slot defines the content displayed within the popover when triggered. It can include a variety of HTML elements, such as descriptive text, images, forms, or other interactive components.
```